app.rb

```diff
@@ -31,6 +31,7 @@ class OpenidConnectRelyingParty < Sinatra::Base
 
     configure :development do
       require 'byebug'
+      set :logging, Logger::DEBUG
```

Comment:
Were we intending to change the log level? Is this related to Rubocop?

Reply:
I was going to leave that in, for development purposes.  I'll just remove it, since I can simply re-add when I want debug output.  Personally, I would prefer to have debug/info outputs for this app, since it is used strictly for testing.

Reply:
> it is used strictly for testing.

These apps are [deployed](https://handbook.login.gov/articles/appdev-cloud-gov-deploy.html), where I don't think we'd want this level of logging.

Reply:
OK, deployed for testing. So, I added it under the :development configuration, along with byebug, and thought it would be useful for other developers.  I'll remove it.

I can't tell what other things need to be resolved, to have this PR approved.